src/Kunstmaan/MediaBundle/DependencyInjection/KunstmaanMediaExtension.php

```diff
@@ -116,6 +116,16 @@ public function prepend(ContainerBuilder $container)
 
         $container->prependExtensionConfig('doctrine', $doctrineGedmoEntityConfig);
 
+        $frameworkExtensionConfig = [
+            'assets'=> [
+                'packages' => [
+                    'asset' => null,
+                    'media' => null,
+                ],
+            ],
+        ];
+        $container->prependExtensionConfig('framework', $frameworkExtensionConfig);
```

Comment:
Not sure if this is the right bundle, but there is no core bundle available. Maybe move to the admin bundle?